src/AzureIoTHub.Portal/Shared/Models/v1.0/LoRaWAN/DeviceModelCommand.cs

```diff
@@ -20,6 +20,8 @@ public class DeviceModelCommand
         /// The command frame in hexa.
         /// </summary>
         [Required(ErrorMessage = "The frame is required.")]
+        [MaxLength(255, ErrorMessage = "The frame should be up to 255 characters long.")]
+        [RegularExpression("^[0-9a-fA-F]{0, 255}$", ErrorMessage = "The frame should only contain hexadecimal characters")]
```

Comment:
```suggestion
        [RegularExpression("^[0-9a-fA-F]{0,255}$", ErrorMessage = "The frame should only contain hexadecimal characters")]
```

Not sur that the space in the length token is a problem ?